src/components/RequestGroups/RequestGroupOverview.vue

```diff
@@ -0,0 +1,174 @@
+                  {{ requestgroup.proposal }}
+                </span>
+              </div>
+            </div>
+          </b-col>
+          <b-col class="p-0">
+            <div class="requestgroup-details">
+              <div :class="requestgroup.state | stateToBsClass('text')"><i :class="requestgroup.state | stateToIcon" />{{ requestgroup.state }}</div>
+              <div>
+                <i class="fa fa-fw fa-calendar" />
+                <span :title="requestgroup.modified | timeFromNow">{{ requestgroup.modified | formatDate }}</span>
+              </div>
+            </div>
+          </b-col>
+        </b-row>
+      </b-col>
+      <b-col md="1" cols="12" class="request-count request-block total-requests">
+        <div class="text-center align-middle" title="Total number of requests">{{ requestgroup.requests.length }}</div>
+      </b-col>
+      <b-col md="1" cols="12" class="request-count request-block text-info pending-requests">
+        <div class="text-center align-middle" title="Number of pending requests">{{ requestgroup | requestStateCount(['PENDING']) }}</div>
+      </b-col>
+      <b-col md="1" cols="12" class="request-count request-block text-danger failed-requests">
+        <div class="text-center align-middle" title="Number of failed requests">
+          {{ requestgroup | requestStateCount(['WINDOW_EXPIRED', 'FAILURE_LIMIT_REACHED']) }}
+        </div>
+      </b-col>
+      <b-col md="1" cols="12" class="request-count request-block text-success completed-requests">
+        <div class="text-center align-middle" title="Number of completed requests">{{ requestgroup | requestStateCount(['COMPLETED']) }}</div>
+      </b-col>
+    </b-row>
+  </b-col>
+</template>
+<script>
+import _ from 'lodash';
+
+import { stateToBsClass, stateToIcon, formatDate, timeFromNow } from '@/util';
+
+export default {
+  name: 'RequestGroupOverview',
+  filters: {
+    requestStateCount: function(requestgroup, states) {
+      let count = 0;
+      for (let request of requestgroup.requests) {
+        if (states.includes(request.state)) {
+          count++;
+        }
+      }
+      return count;
+    },
+    stateToBsClass: function(state, prefix) {
+      return stateToBsClass(state, prefix);
+    },
+    stateToIcon: function(state) {
+      return stateToIcon(state);
+    },
+    formatDate(value) {
+      return formatDate(value);
+    },
+    timeFromNow: function(value) {
+      return timeFromNow(value);
+    }
+  },
+  props: {
+    requestgroup: {
+      type: Object,
+      required: true
+    },
+    proposalLink: {
+      type: Object,
+      required: false,
+      default: () => {
+        return {};
+      },
+      validator: value => {
+        let hasHref = 'href' in value;
+        let hasTo = 'to' in value;
+        if (hasHref && hasTo) return false;
+        if (_.isEmpty(value)) return true;
+        return hasHref || hasTo;
+      },
+      description:
```

Comment:
We mentioned changing these to comments so it's clear that the description doesn't affect behavior.